Reagents.ml

```diff
@@ -35,6 +35,7 @@ module type S = sig
   val (<*>)       : ('a,'b) t -> ('a,'c) t -> ('a, 'b * 'c) t
   val attempt     : ('a,'b) t -> ('a, 'b option) t
   val run         : ('a,'b) t -> 'a -> 'b
+  val get_tid     : unit -> int
```

Comment:
`get_tid` should not be part of reagents API. If you notice, reagents do not talk about threads at all. They are part of the scheduler.